src/inference/Inference.jl

```diff
@@ -124,7 +124,6 @@ end
 # Default Transition #
 ######################
 # Default
-# Extended in contrib/inference/abstractmcmc.jl
```

Comment:
@yebai It's still extended there in line 19 for AdvancedHMC, so I suggest keeping this comment.